web/src/pages/Setting/team.scss

```diff
@@ -0,0 +1,82 @@
+.action-ctn{
```

Comment:
You should remove this, this directory is already pushed in current under a new name